cms/envs/test.py

```diff
@@ -132,6 +132,7 @@
 # This hack disables migrations during tests. We want to create tables directly from the models for speed.
 # See https://groups.google.com/d/msg/django-developers/PWPj3etj3-U/kCl6pMsQYYoJ.
 MIGRATION_MODULES = {app: "app.migrations_not_used_in_tests" for app in INSTALLED_APPS}
+MIGRATION_MODULES['credentials'] = 'app.migrations_not_used_in_tests'
```

Comment:
As you can see the line above that should prevent migrations from running during tests but unfortunately migrations for openedx apps and common are still running. So as in this PR we are adding a user in data migration so running of user data migrations will cause lot of user tests to fail.